src/constants.ts

```diff
@@ -41,3 +41,4 @@ export const MIRROR_NODE_LABEL = "mirror-node-rest";
 export const RELAY_LABEL = "json-rpc-relay";
 export const IS_WINDOWS = process.platform === "win32";
 export const UNKNOWN_VERSION = "Unknown";
+export const EVM_ADDRESSES_BLOCKLIST_FILE_NAME = 'evm-addresses-blocklist.csv'
```

Comment:
This shouldn't be constant here, we should take it from the `local.json`. Otherwise In the future if we change the location there, we have to change it here as well.